sdk/include/xdk/ntddk.template.h

```diff
@@ -118,6 +118,8 @@ typedef PIMAGE_NT_HEADERS32 PIMAGE_NT_HEADERS;
 #elif defined(_M_ARM)
 $include(arm/ke.h)
 $include(arm/mm.h)
+#elif defined(_M_ARM64)
+$include(arm/ke.h)
```

Comment:
There should be `$include(arm64/ke.h)` - you have fixed it in the second commit, however the fix should be in this first commit, otherwise it wouldn't be correct.